lib/routes/publico/ciencias.ts

```diff
@@ -0,0 +1,50 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+import getItems from './items-processor';
+
+export const route: Route = {
+    path: '/ciencias/:subsection?',
+    categories: ['traditional-media'],
+    example: '/ciencias',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: true,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['publico.es/ciencias'],
+            target: '/ciencias',
+        },
+    ],
+    name: 'Ciencias - Público',
```

Comment:
No need to include namespace name in route name
```suggestion
    name: 'Ciencias',
```